repair_quotation_manual_sync/models/stock_move.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2025 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+
+from odoo import api, fields, models
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    invoiceable = fields.Boolean(
+        help="Check this box if the product is billable. "
+        "Only invoiceable products will be included in the quotation.",
+        default=True,
+    )
+    repair_create_sync = fields.Boolean(
+        compute="_compute_sync_flags",
+    )
+    repair_update_sync = fields.Boolean(
+        compute="_compute_sync_flags",
+    )
+    is_sale_confirmed = fields.Boolean(
+        compute="_compute_sale_confirmed",
```

Comment:
```suggestion
        compute="_compute_sale_confirmed",
```
I would also update the compute method to be accordingly 👀